lib/primitives/base-element.js

```diff
@@ -132,10 +165,22 @@ module.exports = function(registry) {
     },
 
     fromCompactRefract: function(tuple) {
-      this.meta = tuple[1];
       this.attributes = tuple[2];
       this.content = tuple[3];
 
+      if (tuple[1]) {
+        Object.keys(tuple[1]).forEach(function (key) {
+          var value = tuple[1][key];
+
+          // TODO: Warning - this may raise a false positive :-(
+          if (value && value.length === 4 && _.isString(value[0]) && _.isObject(value[1]) && _.isObject(value[2])) {
```

Comment:
I'm torn on this one. It might be nice to include it so that the behavior is the same for most cases, otherwise you can't load refract with sourcemaps if you are coming from compact refract.